cmd/dep/integration_test.go

```diff
@@ -50,10 +50,20 @@ func TestIntegration(t *testing.T) {
 
 				// Run commands
 				testProj.RecordImportPaths()
-				for _, args := range testCase.Commands {
-					testProj.DoRun(args)
+
+				var err error
+				for i, args := range testCase.Commands {
+					err = testProj.DoRun(args)
+					if err != nil {
+						if i+1 != len(testCase.Commands) {
```

Comment:
A sylistic nit, but I'd make this one statement `if err != nil && i < len(testCase.commands)-1` to cut back on indentation lvls, and change the message to just "unexpected error raised: $s" (ie ditch the "non-final" bit).